src/main/java/org/broadinstitute/consent/http/service/dao/DraftServiceDAO.java

```diff
@@ -0,0 +1,156 @@
+package org.broadinstitute.consent.http.service.dao;
+
+import com.google.inject.Inject;
+import jakarta.ws.rs.BadRequestException;
+import jakarta.ws.rs.NotAuthorizedException;
+import jakarta.ws.rs.NotFoundException;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import org.broadinstitute.consent.http.db.DraftDAO;
+import org.broadinstitute.consent.http.enumeration.UserRoles;
+import org.broadinstitute.consent.http.models.DraftInterface;
+import org.broadinstitute.consent.http.models.DraftSummary;
+import org.broadinstitute.consent.http.models.FileStorageObject;
+import org.broadinstitute.consent.http.models.User;
+import org.glassfish.jersey.media.multipart.FormDataBodyPart;
+import org.jdbi.v3.core.Jdbi;
+
+public class DraftServiceDAO {
+
+  private final Jdbi jdbi;
+  private final DraftDAO draftDAO;
+  private final DraftFileStorageServiceDAO draftFileStorageServiceDAO;
+
+  @Inject
+  public DraftServiceDAO(Jdbi jdbi, DraftDAO draftDAO,
+      DraftFileStorageServiceDAO draftFileStorageServiceDAO) {
+    this.jdbi = jdbi;
+    this.draftDAO = draftDAO;
+    this.draftFileStorageServiceDAO = draftFileStorageServiceDAO;
+  }
+
+  public void insertDraft(DraftInterface draft)
+      throws SQLException, BadRequestException {
+    jdbi.useHandle(handle -> {
+      handle.getConnection().setAutoCommit(false);
+      try {
+        draftDAO.insert(draft.getName(), draft.getCreateDate().toInstant(),
+            draft.getCreateUser().getUserId(), draft.getJson(), draft.getUUID(),
+            draft.getType().getValue());
+      } catch (Exception e) {
+        handle.rollback();
+        throw new BadRequestException(
+            "Error submitting draft.  Drafts require valid json to be submitted.");
```

Comment:
Is there a reason to not include `e` in this exception? Leaving it out means that the underlying exception is lost, and it may contain useful information.